frontend/public/config.yml

```
@@ -274,14 +275,115 @@ collections:
           - {
               label: 'Thank you image',
               name: 'thank_you_image',
               widget: 'image',
             }
 
+      - label: 'LARP Rules'
+        name: 'larp_rules'
+        file: 'frontend/src/content/pages/larp-rules.md'
+        fields:
+          - {
+              label: 'LARP rules page title',
```

yamllint (1.35.1)

[error] wrong indentation: expected 12 but found 14

(indentation)